framework-utilities/utilities/utilities-core/src/main/java/uk/gov/justice/services/common/converter/ObjectToJsonObjectConverter.java

```diff
@@ -1,34 +1,57 @@
 package uk.gov.justice.services.common.converter;
 
+import static java.lang.String.format;
+
 import uk.gov.justice.services.common.converter.exception.ConverterException;
 
 import java.io.IOException;
 
-import javax.inject.Inject;
+import javax.enterprise.inject.Vetoed;
 import javax.json.JsonObject;
 
 import com.fasterxml.jackson.databind.ObjectMapper;
 
 /**
  * Converts a Pojo to a JsonObject
  */
+@Vetoed
 public class ObjectToJsonObjectConverter implements Converter<Object, JsonObject> {
 
-    @Inject
-    ObjectMapper mapper;
+    private final ObjectMapper mapper;
+
+    /**
+     * @see JsonObjectConvertersProducer
+     * @deprecated Please use the parameterised constructor in your tests. This constructor is never
+     * used in production code as it is always injected and will be removed soon.
+     * <p>
+     * There is now a Producer for CDI injection:
+     */
+    @Deprecated
+    public ObjectToJsonObjectConverter() {
+        mapper = null;
+    }
+
+    public ObjectToJsonObjectConverter(final ObjectMapper mapper) {
+        this.mapper = mapper;
+    }
 
     @Override
     public JsonObject convert(final Object source) {
         try {
-            final JsonObject jsonObject = mapper.readValue(mapper.writeValueAsString(source), JsonObject.class);
+            final String json = mapper.writeValueAsString(source);
+
+            if (json != null) {
```

Comment:
Is this new check going to alter the exception that is throw? I guess it shouldn't actually matter, but just want to confirm.

Reply:
No - it's actually a fix of code that didn't actually work, but the test didn't fail correctly